docs/instruments/ZJSONInstrument_template.json

```diff
@@ -0,0 +1,82 @@
+{
+  "schema": {
+    "meta": {
+      "createdBy": "",
+      "createdOn": "",
+      "createdWith": "",
+      "defaultLanguage": "en-CA",
+      "derivedFrom": "",
+      "importedFrom": "",
+      "includeMetaDataFields": true,
+      "instrumentVersion": "",
+      "lastUpdateOn": "",
+      "longName": {
+          "en-CA": "",
+          "fr-CA": ""
+      },
+      "multilingual": true,
+      "previousVersion": "",
+      "schemaVersion": "",
+      "shortName": ""
```

Comment:
The description in the spec

> The unique name and identifier of the instrument, also the test name in the
database.

implies that the shortName is intended for backend purposes.. so I'm not sure that it should be multilingual.

Maybe it would be a good idea to explicitly name it something like "backendName"

Reply:
yes, it should be the ID so a unique string. what about "uniqueName" or "testName" as that's what we call it in LORIS anyways? i don't know if i like "backendName".